app/controllers/concerns/decidim/proposals/orderable.rb

```diff
@@ -31,10 +29,6 @@ def default_order
           if order_by_votes?
             detect_order("most_voted")
           else
-            if current_organization.show_equity_composite_index?
-              "most_equitable"
-            end
```

Comment:
question: Did you intend to remove this default sorting? It makes sense to pull it out to me, given that the equity piece isn't final, I just didn't see it called out.

Reply:
good question. It prompted me to double-check in standup.